public/editor/scripts/ui/publisher.js

```diff
@@ -313,7 +314,8 @@ Publisher.prototype.updateDialog = function(publishUrl, allowUnpublish) {
   var published = this.dialog.published;
   var unpublishBtn = this.dialog.buttons.unpublish;
   var unpublish = this.handlers.unpublish;
-
+    
```

Comment:
It's unhappy with this line for some reason.  What happens if you remove this blank line?